staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -941,11 +770,11 @@ func (vl *VirtualMachineInstancePresetList) GetListMeta() meta.List {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +k8s:openapi-gen=true
 type VirtualMachine struct {
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+	// +k8s:openapi-gen=false
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-
 	// Spec contains the specification of VirtualMachineInstance created
-	Spec VirtualMachineSpec `json:"spec,omitempty"`
+	Spec VirtualMachineSpec `json:"spec" valid:"required`
```

Comment:
@slintes this line is causing the CI errors.

Reply:
oh thanks, before this it was a merge conflict on .gitignore (which I reverted)